src/renderer/services/thorchain/balances.ts

```diff
@@ -1,14 +1,20 @@
 import { AssetRuneNative } from '@xchainjs/xchain-util'
 
-import { triggerStream } from '../../helpers/stateHelper'
+import { observableState /*, triggerStream*/ } from '../../helpers/stateHelper'
 import * as C from '../clients'
 import { client$ } from './common'
 
+const { get$: reloadBalances$, set: setReload } = observableState<'trigger' | ''>('')
```

Comment:
Maybe just a `boolean` for reload state? `observableState<boolean>(false)`

Reply:
i've added a `reset` callback to the `triggerStream` helper with [this commit](https://github.com/thorchain/asgardex-electron/pull/1268/commits/74cb0a3383e53facf8a0d508598a703edd4bea8e). what do you think ?